bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/Registry.java

```diff
@@ -49,15 +54,15 @@
      *            key of the element
      * @return element or null if no element was found
      */
-    public E get(K key);
+    public @Nullable E get(K key);
```

Comment:
> Isn't be better to set @nonnull to ItemRegistry.getItem() instead of @nullable to all registries?

`getItem()` obviously only exists on ItemRegistry, so I do not understand the question.